docs/developers.md

```diff
@@ -38,6 +38,21 @@ How to import a Gradle project:
 * [How to import Gradle project into IntelliJ IDEA](https://www.jetbrains.com/help/idea/gradle.html) (section "Importing a project
   from a Gradle model").
 
+**Note for Eclipse Users:**
+
+Eclipse needs to have Gradle build the generated source files before it can build the project.
+You can do this by running task `genBuildInfo` before importing the project.
+See [Building from sources](#building-from-sources) for how to run Gradle.
+
+To run the task within Eclipse after importing the logisim-evolution project:
```

Comment:
I think we decided that project name is `Logisim-evolution` (with `L`). (multiple places)

Reply:
I am not sure which is actually correct in some of these places. The reader is literally looking at a lowercase (l) on the screen in Eclipse. But I don't think any confusion will result from using L, so I'll go with that.